app/services/publish_manual_service.rb

```diff
@@ -8,6 +10,8 @@ def initialize(dependencies)
 
   def call
     if versions_match?
+      fetch_tags
+      update_manual_with_tags
```

Comment:
Are there any additional potential problems with this dependency between publishing and tag-fetching?

Are there situations where we'd currently (and/or in the near future) still be able to (and want to) publish while _some_ services are down? (If you see what I mean..?)